### ISSUE TITLE
WCAG: Focus ring has insufficient contrast around certain interactive controls

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#3916]
Spawned from #3892.

"Focus ring (dotted/solid line around the control that has focus) has insufficient contrast around certain interactive controls as user tabs through the page. | Interactive elements do not show visual focus when it receives tab focus. | A keyboard user tabbing through the page will not know when focus is on controls that don't have a focus ring."

![68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3536393531373863363338653731386339393630303131352f39613763356563392d306566382d346635342d613238622d353263623538383737353439](https://user-images.githubusercontent.com/5294596/120225852-04c3b000-c214-11eb-9551-410a0caa62af.png)
![68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3536393531373863363338653731386339393630303131352f39383964616563332d613961662d343236302d616437322d326333326635333933366337](https://user-images.githubusercontent.com/5294596/120225860-068d7380-c214-11eb-8596-30e5ae3e2d8d.png)

"The anchor outline:none appears to over ride the :focus class to remove the visual focus when the tab key is used to access the element.. To solve this, remove outline:none from the anchor classAnd focus:noneNote: if these are user agent Style sheet settings, simply override them with visible outline values in the CSS. In the example solution to the left its a grey outline. It could be and dark color such as red, dark blue, green etc... |   | Delete this {outline: none} This code will ensure the focus is always visible and has sufficient contrast regardless of the background.*:focus { box-shadow: inset 0 0 0 1px black, 0 0 3px 1px white !important;outline: dashed !important;}Live example linked to the right"
